### PR TITLE
AOVs work 1

### PR DESCRIPTION
**Pull request by est77**
- Remove automatic depth AOV.
- Remove some AOV related methods from the python bindings.

### CODE REVIEW

File: src/appleseed/renderer/kernel/rendering/localsampleaccumulationbuffer.cpp [hunk 1]
```diff
@@ -185,7 +185,7 @@ void LocalSampleAccumulationBuffer::store_samples(
 
                 const float fx = s->m_position.x * level_width;
                 const float fy = s->m_position.y * level_height;
-                level->add(fx, fy, s->m_values);
+                level->add(fx, fy, &s->m_color.r);
```
Comment (dictoon) [inline src/appleseed/renderer/kernel/rendering/localsampleaccumulationbuffer.cpp]: `&s->m_color[0]` is more idiomatic.

File: src/appleseed/renderer/kernel/rendering/localsampleaccumulationbuffer.cpp [hunk 2]
```diff
@@ -346,45 +341,40 @@ void LocalSampleAccumulationBuffer::develop_to_tile_undo_premult_alpha(
...
 
             // Quick run.
             for (size_t ix = prefix_end; ix < suffix_begin; ix += m)
             {
-                level.get_pixel(src_base + (ix >> s), &values[0]);
-                const float rcp_alpha = values[3] == 0.0f ? 0.0f : 1.0f / values[3];
-                values[0] *= rcp_alpha;
-                values[1] *= rcp_alpha;
-                values[2] *= rcp_alpha;
+                level.get_pixel(src_base + (ix >> s), &values.r);
```
Comment (dictoon) [inline src/appleseed/renderer/kernel/rendering/localsampleaccumulationbuffer.cpp]: Same thing.

File: src/appleseed/renderer/kernel/rendering/sample.h [hunk 3]
```diff
@@ -32,6 +32,7 @@
 
 // appleseed.foundation headers.
 #include "foundation/math/vector.h"
+#include "foundation/image/color.h"
```
Comment (dictoon) [inline src/appleseed/renderer/kernel/rendering/sample.h]: Sort alphabetically.

**Comment by dictoon** [comment]
Thanks, such a relief to get rid of those depth files!

You didn't change anything in the frozen display renderer. What happens if we try to use it?

**Comment by est77** [comment]
> You didn't change anything in the frozen display renderer. What happens if we try to use it?

True. Probably we crash?
Should I remove it (we could always recover it from the repo history)?

**Comment by dictoon** [comment]
Yes, just remove it for now.

**Comment by est77** [comment]
Fixed comments and removed frozen display renderer.

**Comment by dictoon** [comment]
Thanks. Good to merge once Travis is done.

**Pull request merged**